app/managers.py

```diff
@@ -0,0 +1,26 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connect = sqlite3.connect("../cinema.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        sql = "Insert Into actors (first_name,last_name) VALUES (?,?)"
```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to use uppercase for commands and split query in lines: (
            f"INSERT INTO {self._table_name} "
            "(first_name, last_name) VALUES (?, ?)"
        )
        
 Same with other queries

Reply:
You can also try to rewrite without creating additional variable

Reply:
Is it ok to stay sql variable?